stable/insights-agent/download-kubectl.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/sh
+
+set -e
```

Comment:
I think this needs to be `set -eo pipefail`
Otherwise the pipeline will still not fail